src/test/java/com/google/sps/testData.java

```diff
@@ -0,0 +1,107 @@
+package com.google.sps.servlets;
```

Comment:
This file should start with a capitol T.

To improve readability, you may want to actually make this a class with helper methods, or just move these methods back into the classes they are used by.

Reply:
Fixed file name in b544a05.

What do you mean a class with helper methods? I made this a superclass so that both of my other test files could access the test objects  and method here without having to copy paste them in both.